apps/health_campaign_field_worker_app/integration_test/inventory_package/pages/view_reports.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:digit_components/digit_components.dart'; // Import components from Digit Components package
+import 'package:flutter/material.dart'; // Import Flutter Material library
+import 'package:flutter_test/flutter_test.dart'; // Import Flutter test library
+import 'package:intl/intl.dart'; // Import intl library for date formatting
+
+import '../../test_variables.dart'; // Import test variables and widgets
+
+final testVariables = getTestData(); // Get test data
+final widgetSelector = getWidgets(); // Get widget selectors
+final boundaryNames = getBoundaryName(); // Get boundary names
+
+// Function to test view reports page
+Future<void> testViewReportsPage(WidgetTester widgetTester) async {
+  // Tap on view reports button
+  await widgetTester.tap(widgetSelector['viewReports']!);
+  await widgetTester.pumpAndSettle(const Duration(milliseconds: 1000));
+
+  // Iterate through the pages
+  for (int i = 5; i < 6; i++) {
+    await testOnePage(widgetTester, i); // Test each page
+    // Go back to view reports page
+    await widgetTester.tap(widgetSelector['viewReports']!);
+    await widgetTester.pumpAndSettle(const Duration(milliseconds: 1000));
+  }
```

Comment:
There is only one widget for View Reports, the loop has been added because we go to home page at the end of every report. Hence, no change is required in this regard.